js/events/pointer.js

```diff
@@ -95,6 +96,20 @@ each(EventStrategy.map, function(pointerEvent, originalEvents) {
     registerEvent(pointerEvent, new EventStrategy(pointerEvent, originalEvents));
 });
 
+if(jQuery && support.touch) {
+    each([TouchStrategy.map["dxpointerdown"], TouchStrategy.map["dxpointermove"]], function(_, eventName) {
+        jQuery.event.special[eventName] = {
+            setup: function(_, namespaces, handler) {
+                if(namespaces.indexOf(TouchStrategy.POINTER_EVENTS_NAMESPACE) > -1) {
+                    this.addEventListener(eventName, handler, { passive: false });
```

Comment:
And, please use `domAdapter.listen` instead of `addEventListener`

Reply:
fixed